### ISSUE TITLE
Fails to run on GTK-3

### ISSUE THREAD

**Issue by bhepple** [SeaDve/Kooha#80]
Doesn't run on fedora-33:

```
$ kooha 
Traceback (most recent call last):
  File "/usr/local/bin/kooha", line 31, in <module>
    from kooha import main
  File "/usr/local/share/kooha/kooha/main.py", line 9, in <module>
    gi.require_version('Gtk', '4.0')
  File "/usr/lib64/python3.9/site-packages/gi/__init__.py", line 129, in require_version
    raise ValueError('Namespace %s not available for version %s' %
ValueError: Namespace Gtk not available for version 4.0
```

It would be nice to document the required environment on the front page.

**Comment by SeaDve** [comment]
I'll update readme, but for now, its more recommended to use flatpak as it provides more consistent package versions etc. and you don't have to worry about those

**Comment by SeaDve** [comment]
aur btw has listed the needed dependecies https://aur.archlinux.org/packages/kooha

**Comment by SeaDve** [comment]
fixed with 10fb27f14c4ed10e763100a1ac68f10dbf7ec9f4
